### PR TITLE
[script.module.libwdr] v1.0.0

### PR DESCRIPTION
**Pull request by prof-membrane**
### Description
This is a scraper for http://www1.wdr.de/mediathek/video and websites which use the same framework.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by phil65** [comment]
wouldnt it make sense to merge this module with script.module.libmediathek3 ?

**Comment by prof-membrane** [comment]
I think it is easier to keep every scraper seperated - there are 12 in total. I don't want to pack everything in one monolythic module, but I could arrange that if you want.

The framework is also used by my NHL add-on and might be used by some other unrelated add-ons in the future.

**Comment by phil65** [comment]
I dont really mind, some of our team members have some hostility towards too many module add-ons though :)

**Comment by prof-membrane** [comment]
Every scraper will be used in at least four add-ons. 
1. A seperate "Mediathek" add-on (in this case "WDR Mediathek")
2. A general "Mediathek" add-on which merges content from all the scrapers ("Unithek")
3. An "shortcut" add-on which merges all otherwise seperate Mediathek add-ons to keep the video add-ons list clean.
4. A PVR releated add-on which lets you rewatch past episodes from the EPG.

In case of libwdr, there will be two additional add-ons: "Sportschau" and "WDR Rockpalast". An audio add-on might follow. 

The libs are more or less just API scrapers. They are compatible with projects unreleated to Kodi.


Please confer with your teammembers if I can submit the scrapers in this state. I can merge them if you really want, but that will have some downsites:
- all otherwise seperate libs have to be downloaded in case of a change in one scraper
- the whole script has to be reviewed in case of one minor bugfix (could be mitigated by diff)
- error prone copying from dev- to release-version to fix one scraper
- no possibility of maintainer switching for the modules

A change like this would set me back a week. I want to release the 'most important' add-ons before Krypton, so I want to hurry up. I have 18 add-ons ready, 15 of them are using at least one of the 11 scrapers (other add-ons are planned).

@phil65 Can I make a single PR with a bunch of the add-ons/scripts (packed in different commits)? This would accelerate the release immensely.

File: script.module.libwdr/addon.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.libwdr" 
+	     name="libwdr" 
+	     version="1.0.0" 
+	     provider-name="membrane">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0" />
```
Comment (ronie) [inline script.module.libwdr/addon.xml]: depend on v 2.25.0 please

**Comment by prof-membrane** [comment]
Done.

**Pull request merged**